### PR TITLE
LPD-8481 Remove unused reference from ModulePortalProfile

### PR DESCRIPTION
**Pull request by MrJohn1911**

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:report:2544141127

File: modules/dxp/apps/portal/portal-properties-swapper/src/main/java/com/liferay/portal/properties/swapper/internal/portal/profile/ModulePortalProfile.java [hunk 1]
```diff
@@ -44,7 +43,4 @@ protected void activate(ComponentContext componentContext) {
 	@Reference(target = ModuleServiceLifecycle.PORTAL_INITIALIZED, unbind = "-")
 	private ModuleServiceLifecycle _moduleServiceLifecycle;
 
-	@Reference(target = "(default=true)")
```
Comment (dantewang) [inline modules/dxp/apps/portal/portal-properties-swapper/src/main/java/com/liferay/portal/properties/swapper/internal/portal/profile/ModulePortalProfile.java:47]: @MrJohn1911 Please review the git history to see whether this was added to ensure execution order.

Comment (MrJohn1911) [reply modules/dxp/apps/portal/portal-properties-swapper/src/main/java/com/liferay/portal/properties/swapper/internal/portal/profile/ModulePortalProfile.java:47]: Hey @dantewang I was looking for previous commits that changed this class, and what I found was that the class `ModulePortalProfileEnabler` added a dependency for the `StoreFactory` to solve a bug, but after that, shuyang did a refactor in `StoreFactory` and this class was removed and its logic was moved to `DLStoreImpl`, and the `ModulePortalProfileEnabler` was removed and the reference to the `StoreFactory` was moved to `ModulePortalProfile`, but now, if I remove the reference, the previous bug does not happen anymore.

**Comment by MrJohn1911** [comment]
Hi @dantewang do you want me to investigate it further?

File: modules/dxp/apps/portal/portal-properties-swapper/src/main/java/com/liferay/portal/properties/swapper/internal/portal/profile/ModulePortalProfile.java (hunk 1 above)
Comment (dantewang) [reply modules/dxp/apps/portal/portal-properties-swapper/src/main/java/com/liferay/portal/properties/swapper/internal/portal/profile/ModulePortalProfile.java:47]: @MrJohn1911 Is there a test for the previous bug, or will it be reproduced just by starting the portal? Let's make sure that is correctly covered for this PR. If CI is clean, and every related verification passes, we can move on.

**Comment by tinatian** [comment]
@MrJohn1911 Close, please resend if needed.

Thanks.
Tina

**Pull request closed without merging**